### DISCUSSION TITLE
Powerdns-admin interface, (iface_expire = 0) will lock out the interface, not disable lockout.

### DISCUSSION THREAD

**Discussion by Fingail** [poweradmin/poweradmin#535] [Q&A]
I set the web interface timeout to 0 thinking it would disable the timeout, immediately kicked out of interface and now I can't login because the page keeps refreshing.

I think this is the variable in question:
https://github.com/poweradmin/poweradmin/wiki/Configuration-File
$iface_expire	1800	Session time-out in seconds. After this timeout, you are automagically logged out.

**Comment by Fingail** [comment]
Resolved issue by changing the value of the corresponding value in the setting table in SQL

**Comment by edmondas** [comment]
As I am not sure if having no session timeout at all would be secure, I suggest setting a larger timeout value instead.
